<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>351</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_4">
       <property name="plainText">
        <string>获取选中文本</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_6">
       <property name="plainText">
        <string>设置文本</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_5">
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="plainText">
        <string>获取文本
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="plainText">
        <string>光标位置改变</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="plainText">
        <string>文本被修改
1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>261</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_11">
       <property name="plainText">
        <string>粘贴剪贴板文本</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_9">
       <property name="plainText">
        <string>清除所有文本</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_10">
       <property name="plainText">
        <string>拷贝文本到剪贴板</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_8">
       <property name="plainText">
        <string>在光标处插入文本   1</string>
       </property>
       <property name="cursorWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_12">
       <property name="plainText">
        <string>设置最大行数</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_7">
       <property name="plainText">
        <string>在末尾添加文本</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
